tests/unit/test_cli.py

```diff
@@ -0,0 +1,53 @@
+import click
+from click.testing import CliRunner
+from colin.cli.colin import check, list_checks, list_rulesets
+
+def _call_colin(fnc):
+    runner = CliRunner()
+    return runner.invoke(fnc)
+
+def test_check_command():
+    result = _call_colin(check)
+    expected_output = """Usage: check [OPTIONS] TARGET
+
+Error: Missing argument "target".
+"""
+    assert result.exit_code == 2
+    assert result.output == expected_output
+
+def test_list_checks():
+    result = _call_colin(list_checks)
+    expected_output = """LABELS:
```

Comment:
The output is something, that can be easily changed a bit in the future, but for now, it's ok.